ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -566,15 +576,15 @@ aaa_accounting:
   exec:
     default:
       type: < none | start-stop | stop-only >
-      group: < group_name >
   commands:
     commands_default:
       - commands: < all | 0-15 >
         type: < none | start-stop | stop-only >
-        group: < group_name >
+        group: < group group_name >
```

Comment:
wrong. Here it should just be `group_name`. Variable is `group` so content must be `group_name`

Reply:
Thanks!